proxy/envoy/config.go

```diff
@@ -162,6 +164,12 @@ func buildListeners(env proxy.Environment, sidecar proxy.Sidecar) (Listeners, Cl
 		insertMixerFilter(listeners, instances, sidecar)
 	}
 
+	// Append management listeners after injecting mixer filter to normal listeners
+	// because management port listeners cannot use mixer or auth (see comment
+	// under buildMgmtPortListeners)
+	listeners = append(listeners, mgmtListeners...)
```

Comment:
Not sure I understand. the normalize() calls are still happening before returning the final set of listeners and clusters. I thought the normalize was responsible to deduplicate